cmd/mattermost/commands/config.go

```diff
@@ -217,6 +229,45 @@ func configSetCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func configMigrateCmdF(command *cobra.Command, args []string) error {
+	// Parse source config; defaults to global --config unless overwritten by --from
+	from, err := command.Flags().GetString("from")
+	if err != nil {
+		return errors.Wrap(err, "failed reading source config parameter")
+	}
+	if from == "" {
+		from = viper.GetString("config")
+	}
+
+	// Parse destination config store
+	to, err := command.Flags().GetString("to")
+	if err != nil {
+		return errors.Wrap(err, "failed reading destination parameter (--to)")
+	}
+	if to == "" {
+		return errors.New("Destination DNS required (--to)")
+	}
+
+	// Get source config store - invalid config will throw error here
+	fromConfigStore, err := config.NewStore(from, false)
+	if err != nil {
+		return errors.Wrap(err, "failed to get source config store")
+	}
+	// Get destination config store
+	toConfigStore, err := config.NewStore(to, false)
+	if err != nil {
+		return errors.Wrap(err, "failed to initialize destination config store")
+	}
+
+	// Copy config from source to destination
+	_, err = toConfigStore.Set(fromConfigStore.Get())
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate configuration")
```

Comment:
Propose `config` to keep terminology consistent in this file, at least:
```suggestion
		return errors.Wrap(err, "failed to migrate config")
```